docs/api/vi.md

```diff
@@ -777,3 +777,10 @@ test('Server started successfully', async () => {
 ```
 
 If `vi.useFakeTimers` is used, `vi.waitFor` automatically calls `vi.advanceTimersByTime(interval)` in every check callback.
+
+### vi.waitUntil
+
+- **Type:** `function waitUntil(callback: WaitUntilCallback, options?: number | WaitUntilOptions): Promise`
+- **Version**: Since Vitest 0.34.5
+
+This is similar to `vi.waitFor`, but if the callback throws any errors, execution is immediately interrupted and an error message is received. If the callback returns falsy value, the next check will continue until truthy value is returned. This is useful when you need to wait for something to exist before taking the next step.
```

Comment:
Can we add some code examples for both methods? I think `document.querySelector` is a nice one.

```ts
const element = await vi.waitUntil(() => document.querySelector('.element'))
vi.waitFor(() => {
  expect(document.querySelector('.element')).toBeTruthy()
})
```

Reply:
Added.